docs/web3.eth.rst

```diff
@@ -1282,22 +1282,28 @@ Contracts
 
     The following arguments are accepted for contract class creation.
 
-    - ``abi``
-    - ``asm``
-    - ``ast``
-    - ``bytecode``
-    - ``bytecode_runtime``
-    - ``clone_bin``
-    - ``dev_doc``
-    - ``decode_tuples``
-    - ``interface``
-    - ``metadata``
-    - ``opcodes``
-    - ``src_map``
-    - ``src_map_runtime``
-    - ``user_doc``
-
-    See :doc:`web3.contract` for more information about how to use contracts.
+    :param abi: Application Binary Interface. Usually provided since an ``abi`` is required to interact with any contract.
```

Comment:
@fselmo @pacrob I like the sphinx `param` block! I'm not sure about all of these params, but I took my best guess at as many as I could. Are there any words we can add or change here?

Thanks!